apps/frontend/src/views/Results.vue

```diff
@@ -262,6 +263,11 @@ export default class Results extends mixins(RouteMixin) {
    */
   status_filter: ControlStatus | null = null;
 
+  /**
+   * Determines if we should filter to only waived controls
+   */
+  waived_filter = false;
```

Comment:
Instead change the type of `status_filter` to `ControlStatus | 'Waived' | null = null`.